src/App/__test__/App.test.jsx

```diff
@@ -103,4 +103,13 @@ describe('Authenticated Application routing', () => {
     mountApp.update();
     testCorrectComponentInPath(mountApp, 'DownloadPage', '/download', history);
   });
+
+  test('Edit button appears after expanding an upload on the dashboard', () => {
+    history.push('/dashboard');
+    mountApp.update();
+    expect(mountApp.find('.editUploadBtn')).toHaveLength(0);
+    mountApp.find('Caret').first().simulate('click');
+    mountApp.update();
+    expect(mountApp.find('.editUploadBtn')).toHaveLength(1);
+  });
```

Comment:
Would this test be more suitable to go into the `dashboard.test.jsx` file?

Reply:
Good point, before I learned that history.push() functions doesn't change history in the jest/enzyme test I planned to make the test one that checked if the page redirected to uploadPage. 

Given the restriction, it makes sense to move and has been done now.